HTShowcase/HTShowcase/ViewControllers/ShowcaseViewController.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  ShowcaseViewController.swift
+//  HTShowcase
+//
+//  Created by Gonzalo Barrios on 5/22/18.
+//  Copyright © 2018 Gonzalo Barrios. All rights reserved.
+//
+
+import UIKit
+import RxSwift
+import RxCocoa
+
+enum ViewLayout {
+    case common
+    case squared
+    case highImage
+}
+
+enum LayoutIconImageName: String {
```

Comment:
You can declare a function inside the layout that returns the image name for all the enums. I think it needs to be static but you can do it